src/Core/Services/AuthService.cs

```diff
@@ -352,7 +352,7 @@ private async Task<SymmetricCryptoKey> MakePreloginKeyAsync(string masterPasswor
             {
                 await _tokenService.SetTwoFactorTokenAsync(tokenResponse.TwoFactorToken, email);
             }
-            await _tokenService.SetTokensAsync(tokenResponse.AccessToken, tokenResponse.RefreshToken);
+            await _tokenService.SetAccessTokenAsync(tokenResponse.AccessToken, true);
```

Comment:
Won't there be a problem by not setting the `RefreshToken` there? Or is that planned to be moved to another place to be saved? Because other than in `ApiService -> DoRefreshTokenAsync` I don't see any other place where that's being saved with this change

Reply:
Both tokens are properly "set" on the next line (new `Account` constructor):

```
new Account.AccountTokens()
{
    AccessToken = tokenResponse.AccessToken,
    RefreshToken = tokenResponse.RefreshToken,
}
```

...and are then immediately available via `StateService`, which is used to perform addition get/set operations on the RefreshToken within `TokenService`.  The only reason I couldn't remove this particular line outright is because `TokenService` requires an in-memory copy of the AccessToken to decode the values necessary for the other parts of the Account constructor (userId, email).